common/executor.go

```diff
@@ -70,4 +72,6 @@ type Executor interface {
 	Claim(swapId common.Hash, randomNumber common.Hash) (string, *Error)
 	// Refund sends refund tx
 	Refund(swapId common.Hash) (string, *Error)
+	// SendAmount
+	SendAmount(address string, amount *big.Int) (string, error) // TODO should be *Error?
```

Comment:
```suggestion
	SendAmount(address string, amount *big.Int) (string, error)
```

Can remove this comment